src/main/resources/dbmigration/mysql/V3__createTables.sql

```diff
@@ -0,0 +1,81 @@
+drop table user;
+
+CREATE TABLE `user` (
+  `userId` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `firstName` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  `lastName` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  `note` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  `virtualBalance` decimal(14,2) NOT NULL DEFAULT '0.00',
+  `winnings` decimal(14,2) NOT NULL DEFAULT '0.00',
+  `status` enum('ACTIVE','BLACKLISTED') CHARACTER SET utf8 COLLATE utf8_bin NOT NULL,
+  `modified` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `created` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`userId`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `tournament` (
```

Comment:
know its a small thing, but i like to follow the camelcase convention :) can we please make it "Tournament"